### ISSUE TITLE
NVDA does not announces complete information after user Input “3000” as  local port and press enter

### ISSUE THREAD

**Issue by Yash14j** [microsoft/vscode#129957]
**GitHubTags:**#A11y_VSCode;#A11ySev2;#A11yTCS;#DesktopApp;#Visual Studio Code Client;#WCAG1.3.1;#DesktopApp;#Win10;#VisualStudioCode-Win32-Apr2021;#A11yMAS;#FTP;#STP;#JAWS;#NVDA;

### Actual:
When NVDA is on and user Input “3000” as  local port and press enter, NVDA does not announces complete information. It only announces "Quick Input list Launch via NPM 1 of 1" and does not announces name of combo box.

**Similar issue is observed with JAWS

Similar issue is observed in Connect to Kubernetes-Select Yes No screen**

### Expected:

When NVDA is on and user Input “3000” as  local port and press enter, NVDA should announce complete information. It should announce combobox chosse the launch configuration to use to run tour component locally  Quick Input list Launch via NPM 1 of 1
### 
Environment Details:
Application Name: Bridge to Kubernetes (Mindaro)
Windows Version: Windows10

### Repro Steps:

1. Setup The Kubernetes extension.
2. Open visual studio code and load "Bikes" project.
3. Tab to "View" menu item in top navigation bar, press enter to expand and use down arrow key to Select "Command Pallet" option.
4. Search and select "Local Process with Kubernetes: Configure Local Process with Kubernetes”
5. Connect to Kubernetes panel will open.
6. Tab to bike options and press enter
7. Start the NVDA 
8. Input “3000” as your local port and press enter to check the result
### 
User Impact:
Screenreader user will not be able to get complete information about combo edit and dropdown

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!
